packages/components/Menu/src/MenuItemCheckbox/MenuItemCheckboxTokens.macos.ts

```diff
@@ -7,7 +7,7 @@ import type { MenuItemCheckboxTokens } from './MenuItemCheckbox.types';
 export const defaultMenuItemCheckboxTokens: TokenSettings<MenuItemCheckboxTokens, Theme> = (t: Theme): MenuItemCheckboxTokens => ({
   backgroundColor: t.colors.transparentBackground,
   borderRadius: 5,
-  checkmarkPadding: globalTokens.sizeNone,
+  checkmarkPadding: globalTokens.size20,
```

Comment:
Do we need any changes to `MenuItemRadioTokens` too?

Reply:
Not for this bug, and not that I can tell for sure.
I took that value precisely from `MenuItemRadioTokens`.

Other properties also differ between these controls but I don't know if they are justified to do so.